config.yaml

```diff
@@ -3,6 +3,9 @@
 # data.gouv.fr API base URL
 datagouvfr_api_url: https://www.data.gouv.fr/api
 
+frontend_setttings:
```

Comment:
typo in settings (I would just remove the term, everything in config.yaml is a setting anyway)